src/Foundation.Find.Cms/Config/FacetFiltersConfigurationProperty.cs

```diff
@@ -0,0 +1,14 @@
+﻿using EPiServer.PlugIn;
+
+namespace Foundation.Find.Cms.Config
```

Comment:
Change namespace to Foundation.Find.Cms.Facets.Config

Reply:
Done